base_classes/NXroot.nxdl.xml

```diff
@@ -83,14 +83,20 @@
     </group>
     <attribute name="default">
         <doc>
+            .. index:: find the default plottable data
             .. index:: plotting
-            
-            Declares which :ref:`NXentry` group contains 
+            .. index:: default attribute value
+
+            Declares which :ref:`NXentry` group contains
             the data to be shown by default.
-            It is needed to resolve ambiguity when 
-            more than one :ref:`NXentry` group exists. 
-            The value is the name of the default :ref:`NXentry` group.
-            
+            It is needed to resolve ambiguity when
```

Comment:
suggest changing "needed" to "used" to back up the idea that the attribute is optional.

Reply:
resolved per 67c4ea7